plugins/TopPairMuonPlusJetsSelectionFilter.cc

```diff
@@ -367,13 +367,11 @@ void TopPairMuonPlusJetsSelectionFilter::goodIsolatedMuons() {
 		if (!( muon.isGlobalMuon() || muon.isTrackerMuon() ) )
 			continue;
 
-		// bool passesIso = getRelativeIsolation(muon, 0.4, useDeltaBetaCorrectionsForMuons_) < tightMuonIso_;
 		bool passesIso = false;
-
         if ( nonIsolatedMuonSelection1_ ) {
         	passesIso = getRelativeIsolation(muon, 0.4, true) > controlMuonIso1_;
         }
-        if ( nonIsolatedMuonSelection2_ ) {
+        else if ( nonIsolatedMuonSelection2_ ) {
```

Comment:
Can you fix the indentation around here?  If read it twice now and thought there was a bug.